extra/modules/fiftyone-devicedetection/src/main/java/org/prebid/server/hooks/modules/fiftyone/devicedetection/model/config/AccountFilter.java

```diff
@@ -0,0 +1,17 @@
+package org.prebid.server.hooks.modules.fiftyone.devicedetection.model.config;
+
+import lombok.Data;
+
+import java.util.List;
+
+/**
+ * Config fragment to restrict module usage by requesting accounts.
+ */
+@Data
+public final class AccountFilter {
+    /**
+     * Allow-list with account IDs.
+     * @see org.prebid.server.settings.model.Account#getId()
```

Comment:
Same for here, and also check for other occurrences.

Reply:
✔️ Applied